### ISSUE TITLE
Sound Classification TM Documentation Page

### ISSUE THREAD

**Issue by shiffman** [ml5js/ml5-website-v02-docsify#174]
I recall an issue about this but I couldn't find it so am opening one! Please close if it's duplicate!

Just noting that we should prepare a documentation page for Sound + Teachable Machine on the website to go along with [Image + Teachable Machine](https://docs.ml5js.org/#/reference/image-classifier-tm)

💜 💜 💜 💜 💜

**Comment by MOQN** [comment]
Yes! The web team will kick off soon, too! We will address as soon as possible.

**Comment by alanvww** [comment]
Thanks, Dan! I was planning to create a issue and discuss this during the web team kickoff meeting, as it’s on my to-do list.

This is also mentioned in [this issue](https://github.com/ml5js/ml5-next-gen/issues/199#issuecomment-2334512132).
